.github/workflows/ruby_tests.yml

```diff
@@ -123,6 +123,10 @@ jobs:
         if: github.event_name != 'push'
         run: |
           bundle exec rake test:foreman_puppet
+      - name: Run access permission test
+        if: ${{ startsWith(matrix.foreman-core-branch, '3') && github.event_name != 'push' }}
```

Comment:
Is this code currently reachable?

Reply:
The if condition is being checked, but the test does not run as we do not have a foreman core branch starting with '3' (, yet).